### PR TITLE
Lu 6623 sqs removal

### PR DESCRIPTION
**Pull request by KrisRogos**
Changes:
* Removed SQS
* Replaced save/get/read functions with appropriate alternatives
* Introduced an `aggregation_files` variable instead of relying on SQS messages for file names
* Replaced `first_agg`, `second_agg`, `third_agg` and their `..._df` counterparts with meanigfull names
* Updated tests and documentation to match.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ e.g. {"success": True/False, "checkpoint"/"error": 4/"Message"}
 
 #### Combiner
 
-The combiner is used to join the outputs from the 3 aggregations back onto the original data. It is assumed that the imputed(or original if it didnt need imputing) data is stored in an s3 bucket by the imputation module; and that each of the 3 aggregation processes each write their output to sqs. <br>
-The combiner merely picks up the imputation data from s3, then 3 messages from the sqs queue. It joins these all together and sends onwards. The result of which is that the next module(disclosure) has the granular input data with the addition of aggregations merged on.
+The combiner is used to join the outputs from the 3 aggregations back onto the original data. It is assumed that the imputed(or original if it didnt need imputing) data is stored in an s3 bucket by the imputation module; and that each of the 3 aggregation processes each write their output to S3. <br>
+The combiner merely picks up the imputation data and the 3 files from the other aggregation stages from s3. It joins these all together and sends onwards. The result of which is that the next module(disclosure) has the granular input data with the addition of aggregations merged on.
```
Comment (Piwington) [inline README.md:78]: Should probably say 'for the other' rather than 'from the other'

File: combiner.py [hunk 2]
```diff
@@ -32,11 +30,9 @@ def handle_error(self, e, data, **kwargs):
     additional_aggregated_column = fields.Str(required=True)
     aggregated_column = fields.Str(required=True)
     in_file_name = fields.Str(required=True)
-    location = fields.Str(required=True)
+    aggregation_files = fields.Dict(required=True)
```
Comment (Piwington) [inline combiner.py:33]: Can this be bumped one line higher to make alphabetical?

File: combiner.py [hunk 3]
```diff
@@ -78,83 +74,55 @@ def lambda_handler(event, context):
         additional_aggregated_column = runtime_variables["additional_aggregated_column"]
         aggregated_column = runtime_variables["aggregated_column"]
         in_file_name = runtime_variables["in_file_name"]
-        location = runtime_variables["location"]
+        aggregation_files = runtime_variables["aggregation_files"]
```
Comment (Piwington) [inline combiner.py:77]: As above.

File: README.md (hunk 1 above)
Comment (KrisRogos) [reply README.md:78]: The combiner is reading the file produced by the other aggregations, so I'd think 'from' makes more sense. 'For' would imply that it is doing something they would then use.

**Pull request merged**